src/theme/components/Divider.ts

```diff
@@ -0,0 +1,6 @@
+// TODO: Figure out why this does nothing
+export const Divider = {
+  defaultProps: {
+    borderColor: "rgba(0, 124, 253, 0.15)",
```

Comment:
I'm gonna follow this up in another branch, just want to get the theme config into main so @MrArnoldPalmer can work with the updated styles